### ISSUE TITLE
it's very hard to resize this, and on mobile and web the size is different for the non-icon one

### ISSUE THREAD

**Issue by jeffreyliu8** [Skyost/DayNightSwitcher#2]
it's very hard to resize this, and on mobile and web the size is different for the non-icon one

**Comment by Skyost** [comment]
> it's very hard to resize this

Agreed. For you now you could use something like a [`Transform`](https://api.flutter.dev/flutter/widgets/Transform/Transform.scale.html) widget.

> on mobile and web the size is different for the non-icon one

It should be the same as the width [is fixed](https://github.com/Skyost/DayNightSwitcher/blob/master/lib/src/widgets.dart#L111). Could please tell me how you did measure that ?

**Comment by jeffreyliu8** [comment]
I tried transform. Scale, but the size change with lots of placeholder empty space. Yes you are right on the width part. I tried to put this widget inside an appbar action, and tried to scale it with viewing in chrome browser.
